desktop/ui/src/main/java/org/datacleaner/widgets/DCGroupComboBox.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.widgets;
+
+import java.awt.Font;
+
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.DefaultListCellRenderer;
+import javax.swing.JLabel;
+import javax.swing.JList;
+
+import org.apache.metamodel.util.HasName;
+import org.datacleaner.util.StringUtils;
+
+/**
+ * Defines a group combo-box class that has a more convenient listening mechanism.
+ */
+public class DCGroupComboBox extends DCComboBox<HasName> {
+    private static final String GROUP_ITEM_INDENTATION = "  ";
+    private boolean _groupExists = false;
+
+    private static class Delimiter implements HasName {
+        private final String _text;
+
+        private Delimiter(String text) {
+            _text = text;
+        }
+
+        @Override
+        public String toString() {
+            return _text.toString();
+        }
+
+        @Override
+        public String getName() {
+            return _text;
+        }
+    }
+
+    public DCGroupComboBox() {
+        super(new GroupComboBoxModel());
+        setRenderer(new GroupComboBoxRenderer());
+    }
+
+    public void addDelimiter(String text) {
+        _groupExists = true;
+        addItem(new Delimiter(text));
+    }
+
+    private static class GroupComboBoxModel extends DefaultComboBoxModel {
```

Comment:
The serializable class GroupComboBoxModel does not declare a static final serialVersionUID field of type long.
DefaultComboBoxModel is a raw type. References to generic type DefaultComboBoxModel<E> should be parameterized.